drivers/bridge/bridge.go

```diff
@@ -562,6 +562,11 @@ func (d *driver) CreateEndpoint(nid, eid types.UUID, epInfo driverapi.EndpointIn
 	}
 	endpoint.macAddress = mac
 
+	// Up the host interface after finishing all netlink configuration
+	if err := netlink.LinkSetUp(host); err != nil {
```

Comment:
Yes, This call works in centos 6.6 because that is how we bring up the container interface inside the sandbox anyways and that works in centos 6.6